lib/floe/workflow/runner.rb

```diff
@@ -5,17 +5,31 @@ class Workflow
     class Runner
       include Logging
 
-      TYPES         = %w[docker podman kubernetes].freeze
       OUTPUT_MARKER = "__FLOE_OUTPUT__\n"
 
       def initialize(_options = {})
       end
 
       class << self
-        attr_writer :docker_runner
+        # see Floe.set_runner
+        def set_runner(name_or_instance = nil, options = {})
+          @docker_runner =
```

Comment:
@kbrock I think leaving the `docker` part out of this is confusing, because we expect to have other types of runners (e.g. lambda, miq/internal, ansible).  I agree it is confusing that there is a docker-docker runner but I think we still need to call out the fact that we're setting a runner for a particular scheme (in this case `docker://`).

Reply:
that is available at https://github.com/ManageIQ/floe/pull/153